src/EnergyPlus/DXCoils.cc

```diff
@@ -5434,16 +5115,14 @@ namespace DXCoils {
                 ErrorsFound = true;
             } else {
                 {
-                    auto const SELECT_CASE_var(GetCurveType(indexSHCurve));
-
-                    if (SELECT_CASE_var == "QUADRATIC") {
+                    if (CurveManager::PerfCurve(indexSHCurve).ObjectType == "Curve:Quadratic") {
```

Comment:
@mjwitte This is one place that seems to rely on a specific curve type. I'm handling this manually for now, but we may want to figure out a more general approach here. I'd need to dig in a bit more to see exactly how these coefficients are being used first.

Reply:
@mjwitte and this comment.

Reply:
For now, I would just leave this requiring quadratic and post a new issue to clean that up so it uses `CurveValue` instead of grabbing and using individual coefficients. If that misses the I/O deadline, so be it.